third_party/xla/xla/tests/BUILD

```diff
@@ -990,6 +991,7 @@ xla_test(
         # TODO(b/151340488): Timed out on 2020-03-12.
         "nozapfhahn",
         "test_xla_cpu_thunks",
+        "cuda-only", #TODO(rocm): weekly sync 24-10-01
```

Comment:
only failed subtests instead of the whole test.

Reply:
Same as above